.jvmopts

```diff
@@ -0,0 +1 @@
+-Djts.overlay=ng
```

Comment:
Using overlayNG makes JTS methods such as union and intersection more robust. In [future versions](https://github.com/locationtech/jts/blob/master/USING.md#jts-system-properties), this setting will become the default.

Reply:
It has to be set prior to starting the JVM. I would suggest to add this as a recommended configuration in the documentation.

Reply:
This is a very good point @thomas-maschler. I will create a separate PR for adding the docs.
If I understood this correctly, this option is applied for the unit tests now. A user installing Mosaic needs to add this option to the env variables in Databricks in order to change the behaviour to `-Djts.overlay=ng`. So this won't be a breaking change.
Am I right @thomas-maschler and @milos-colic ?